09/test.md

```diff
@@ -0,0 +1,244 @@
+
+> Answer:
+> Image ->
+> Audio ->
+> Video ->
+> External Resources (maps, web sites) ->
+
+- What attributes do we use to resize an image tag?
+
+> Answer:
+
+- How can we set fallback to any media tag?
+
+> Answer:
+
+- What is a link? Write an example
+
+> Answer:
+
+- How can we link sections in our document?
+
+> Answer:
+
+- Why do we use `_blank` value?
+
+> Answer:
+
+
+---
+
+# Prueba HTML
+
+- ¿Qué significa HTML?
+
+> Respuesta:  Significa lenguaje de marcado de hipertexto
+
+- ¿Por qué usamos HTML5?
+
+> Respuesta: Para definir la estructura, el diseño y poder ver el contenido de una pagina web
+
+- Enlista las tres etiquetas principales que están en cada documento de HTML y su uso
+
+> Respuesta: `<html doctype>` `<head>` y la etiqueta `<body>`. El doctype es para saber con cual html vamos a trabajar , el head para poner las etiquetas como metas o de especificaciones de la pagina, keywords o descripcion de la pagina y el body para estructuracion del contenido de la pagina.
+
+- Enlista al menos 3 meta tags con su uso y su sintaxis
+
+> Respuesta:`<h1>`para poner algun emcabezado `<ol>`para hacer un lista en orden y `<img>` para añadir una imagen al documento
+
+- ¿Cuántas etiquetas `<h>` existen?
+
+> Respuesta: Desde el `<h1>`hasta el `<h6>`
+
+- ¿Cuál es la diferencia entre `<h>` y `<p>`?
+
+> Respuesta: El `<h>` se usa par los emcabezados y el `<p>`se usa para agregar parrafos
+
+- ¿Cuál es la diferencia entre `<p>` y `<span>`?
+
+> Respuesta: El `<p>`es para poner un parrafo y el `<span>`para dar estilo
+
+- Enlista al menos 2 etiquetas usadas para modificar el estilo del texto
+
+> Respuesta: `<strong>`-`<em>` y `<span>`
+
+- Enlista todas las etiquetas de contenedores vistas en el curso
+
+> Respuesta: `<header>`-`<nav>`-`<section>`-`<aside>`-`<article>`-`<footer>`
+
+- ¿Qué es HTML semántico?
+
+> Respuesta: Es una estructura de codigo para organizar una pagina web de manera que sea mas legible
+
+- ¿Cuál es la diferencia entre `<ul>` y `<ol>`
+
+> Respuesta: `<ul>`para una lista sin orden y `<ol>`para una lista ordenada
+
+- ¿Qué es un input?
+
+> Respuesta: Los inputs se usan para crear controles ,botones o  para obtener informacion de un usuario en un formulario
+
+- Escribe un ejemplo de los siguientes inputs
+
+> Respuesta:
+> Checkbox -> `<input type="checkbox">`
+> Date -> `<input type="Date">`
+> File -> `<input type="file>`
+> Password -> `<input type="password"`
+> Radio -> `<input type="radio">`
+> Range -> `<input type="range">`
+> Text -> `<input type="text"`
+> Textarea -> `<textarea>Escribe</textarea>`
+> Select -> `<select><option>Opcion1</option><option>Opcion2</option></select>`
+
+- ¿Cómo podemos poner un valor por defecto en cualquier input?
+
+> Respuesta: el valor checked o value
+
+- ¿Cuál es el uso de la etiqueta `<label>`?
+
+> Respuesta: darle un nombre a nuestro input
+
+- ¿Cómo hacemos un link entre un input y su label?
+
+> Respuesta: Con el ID del label y del input
```

Comment:
the label tag needs another attribute
La etiqueta label requiere de otro atributo